compiler/runtime/Helpers.inc

```diff
@@ -353,6 +353,8 @@ SETVAL(TR_PPCAESDecrypt, TR_FSRH+86)
 SETVAL(TR_PPCVectorLogDouble, TR_FSRH+87)
 SETVAL(TR_PPCquadWordArrayCopy_vsx, TR_FSRH+88)
 SETVAL(TR_PPCforwardQuadWordArrayCopy_vsx, TR_FSRH+89)
+SETVAL(TR_PPCpostP10ForwardCopy, TR_FSRH+90)
+SETVAL(TR_PPCpostP10GenericCopy, TR_FSRH+91)
```

Comment:
I haven't followed any of the review discussion up to this point.  Is the intended behaviour here to alias the `PPCLEArrayCopy` and `PPCforwardLEArrayCopy` entries because that's how you have written it.

Reply:
no.  in the PR message:  I am recycling the two helpers for LEArrayCopy (I haven't found any use
of them anywhere) for these. But, in order not to break the build, I am
redefining P10 helper indices over LEArrayCopy, before I can delete them.

Reply:
OK, just making sure...